server/route.go

```diff
@@ -236,6 +242,7 @@ func buildRoutes(app *applicationHandler) http.Handler {
 			eventRouter.Use(rateLimitByGroupID(app.limiter))
 			eventRouter.Use(requirePermission(auth.RoleUIAdmin))
 
+			eventRouter.With(rateLimitByGroup(), instrumentPath("/events")).Post("/", app.CreateAppEvent)
```

Comment:
Don't use `rateLimitByGroup()` use this one `appRouter.Use(rateLimitByGroupID(app.limiter))`
```suggestion
			eventRouter.With(rateLimitByGroupID(app.limiter), instrumentPath("/events")).Post("/", app.CreateAppEvent)
```